pilot/pkg/bootstrap/validation.go

```diff
@@ -39,6 +39,8 @@ func (s *Server) initConfigValidation(args *PilotArgs) error {
 		return err
 	}
 
+	s.readinessFlags.configValidationReady = true
```

Comment:
I think this is racy, should be atomic bool?

Reply:
revised, thanks